src/Magnum/OvrIntegration/HmdEnum.h

```diff
@@ -184,6 +211,10 @@ enum class ErrorType: Int {
     MismatchedAdapters = ovrError_MismatchedAdapters,   /**< Chosen rendering adapters between client and service do not match */
     LeakingResources = ovrError_LeakingResources,       /**< Calling application has leaked resources */
     ClientVersion = ovrError_ClientVersion,             /**< Client version too old to connect to service */
+    OutOfDateOS = ovrError_OutOfDateOS,                 /**< The operating system is out of date. */
+    OutOfDateGfxDriver = ovrError_OutOfDateGfxDriver,   /**< The graphics driver is out of date. */
+    IncompatibleGPU = ovrError_IncompatibleGPU,         /**< The graphics hardware is not supported */
+    NoValidVRDisplaySystem = ovrError_NoValidVRDisplaySystem,   /**< No valid VR display system found. */
```

Comment:
`IncompatibleOs`, `OutOfDateOs`, `IncompatibleGpu`, `NoValidVrDisplaySystem`, `UsbBandwidth`, `UsbEnumeratedSpeed` and `HmdFirmwareMismatch` so we have everything CamelCase to be consistent with the rest of Magnum. Looks like even Oculus can't get it consistent in a single enum :)